src/js/hooks/inboundV2/send/useInboundSendForm.js

```diff
@@ -341,6 +382,24 @@ const useInboundSendForm = ({ previous }) => {
     });
   };
 
+  const handleExportFile = async (doc) => {
+    const isValid = await trigger();
+    if (!isValid || !doc?.uri) {
+      return;
+    }
+    await onSave({ showNotification: false });
+    window.open(doc.uri, '_blank');
```

Comment:
Isn't there any more "react" way of doing that? - just out of curiosity. If not, maybe pack the logic into a hook for handling the window object operations?